loyalty_incompatibility/views/loyalty_program_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="loyalty_program_view_form" model="ir.ui.view">
+        <field name="inherit_id" ref="loyalty.loyalty_program_view_form" />
+        <field name="model">loyalty.program</field>
+        <field name="arch" type="xml">
+            <field name="currency_id" position="before">
+                <field
+                    name="incompatible_promotion_ids"
+                    invisible="program_type in ['promotion', 'buy_x_get_y', 'loyalty']"
```

Comment:
```suggestion
                 
```
We can consider that all promotions may have incompatibilities, which could be eliminated as an improvement.